### PR TITLE
Feature/horizontal scrolling

### PR DESCRIPTION
**Pull request by donnapep**

### CODE REVIEW

**Comment by donnapep** [comment]
@rodrigopavezi @stulees Please review. Thx.

File: src/widget/horizontal-scroll.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  function _initScroller() {
+    var title = "",
+      author = "",
+      date = "",
+      story = "",
+      item = null,
+      items = [],
+      scrollerElem = document.querySelector("#scroller");
+
+    _scroller = new RiseVision.Common.Scroller(prefs, params);
```
Comment (stulees) [inline src/widget/horizontal-scroll.js]: @donnapep The `params` already contains `width` and `height` values based on width and height values of `prefs`. I see that the scroller module expects `prefs` due to this line https://goo.gl/U4bqA4 , but I think it's better to leverage just the `params.width` and `params.height` and omit needing to pass prefs. Can we change that?

**Comment by donnapep** [comment]
@stulees Forgot about that. Made the change. Please review. Thx.

**Comment by stulees** [comment]
@donnapep :+1:

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review last commit, thanks

**Comment by donnapep** [comment]
:+1:

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review latest commit. Thx.

**Comment by stulees** [comment]
@donnapep :+1: to your latest commit, can you review my latest commit also

**Comment by donnapep** [comment]
@stulees Yup, looks good. :+1:

**Pull request merged**